experiments/rust_leaf_bench/matched_c/main.c

```
@@ -253,6 +293,19 @@ static void splitmix_fill(SplitMix64 *rng, uint8_t *dst, size_t len) {
   }
 }
 
+static uint64_t load_be64_unaligned(const uint8_t *src) {
+  uint64_t v = 0;
```

Clang (14.0.6)

[warning] variable name 'v' is too short, expected at least 3 characters

(readability-identifier-length)